src/endpoints/stable-diffusion.js

```diff
@@ -412,8 +409,18 @@ comfy.post('/models', jsonParser, async (request, response) => {
         }
         /** @type {any} */
         const data = await result.json();
-        return response.send(data.CheckpointLoaderSimple.input.required.ckpt_name[0].map(it => ({ value: it, text: it })));
-    } catch (error) {
+
+        const ckpts = data.CheckpointLoaderSimple.input.required.ckpt_name[0].map(it => ({ value: it, text: it })) || [];
+
+        // load list of GGUF unets from diffusion_models if the loader node is available
+        const ggufs = data.UnetLoaderGGUF?.input.required.unet_name[0].map(it => ({ value: it, text: `GGUF: ${it}` })) || [];
```

Comment:
You need to do optional chaining through the entire line.

```suggestion
        const ggufs = data.UnetLoaderGGUF?.input?.required?.unet_name?.[0]?.map(it => ({ value: it, text: `GGUF: ${it}` })) || [];
```

Reply:
this safer but if they have UnetLoaderGGUF the other stuff will be there, and if they don't it'll wind up at || []

Reply:
Okay, I see what you did there.